gammapy/irf/psf/map.py

```diff
@@ -523,6 +534,129 @@ def peek(self, figsize=(12, 10)):
         self.exposure_map.reduce_over_axes().plot(ax=axes[2], add_cbar=True)
 
         axes[3].set_title("Containment radius at 1 TeV")
-        self.containment_radius_map(energy_true=2 * u.TeV).plot(
+        kwargs = {self.energy_name:2 * u.TeV}
+        self.containment_radius_map(**kwargs).plot(
             ax=axes[3], add_cbar=True
         )
+
+
+class PSFMapReco(PSFMap):
```

Comment:
I understand correctly, once we have the `PSFMap.energy_axis_name` attribute we don't really need the `PSFMapReco` class except for documentation. I this correct? In this case we could think about just to extend the current documentation of `PSFMap`. and not introduce a new class. What do you think @QRemy  and @LauraOlivera?

Reply:
I see your point @adonath and I don't really have a strong opinion.

I think it would be good to unify both and remove code duplication, but also this PR is almost ready and we do have the deadline of the feature freeze... so I would rather this go in as is than need to re-do it and miss the cut. 

What do you think, @QRemy?

Reply:
> so I would rather this go in as is than need to re-do it and miss the cut.

I think the effort is limited here. Handling both cases with the `PSFMap` will require adapting the docstrings and renaming the argument name. That's basically it...

Reply:
OK, looking more closely the issue that we use the hard-coded axes names for IRFs, like https://github.com/gammapy/gammapy/blob/master/gammapy/irf/psf/map.py#L70.

So handling both cases with the `PSFMap`, would require a new pattern for allowed axes names, which deviated from put current coding pattern. So lets keep the `PSFMapReco`! However we should rename, because our current scheme of naming Classe puts the base class name at the end. Our convention uses "energy" as default for reconstructed energy, however an `EnergyPSFMap` sounds weird. So maybe the best compromise is something like `EnergyRecoPSFMap`. But @registerrierm should comment once again...